### ISSUE TITLE
An error?

### ISSUE THREAD

**Issue by norikaisa** [shap/shap#388]
In the figure of RAD and RM. You write: "In this case coloring by RAD (index of accessibility to radial highways) highlights that the average number of rooms per house has less impact on home price for areas close to radial highways."

I think, it should be: "In this case coloring by RAD (index of accessibility to radial highways) highlights that the average number of rooms per house has less impact on home price for areas far from the radial highways."

**Comment by slundberg** [comment]
Thanks! I realized I actually don't know if a high value means close or far, so I have not made any assumptions in the new text.
